helm/thingsboard/values.yaml

```diff
@@ -17,7 +17,7 @@
 global:
   image:
     server: docker.io
-    tag: "3.4.1"
+    tag: "3.5.0"
```

Comment:
Needed?

Reply:
we have tested this with the latest version of TB, so we want to enforce the version and remove it from anywhere else

Reply:
While we're at it, can we use 3.5.1 (it has a long list of bug fixes (again))

Reply:
done

Reply:
Resolving this, as my question is addressed, please reopen if needed.